admin/schema_changes/13.sql

```diff
@@ -0,0 +1,8 @@
+BEGIN;
+
+ALTER TABLE "comment_revision"
```

Comment:
Generally, we'd have to think about what we want to do with empty comments already in the db before we add a constraint, but I checked and there aren't any empty comments (or any comments whatsoever) in the db so we should be okay.